### DISCUSSION TITLE
Customise the Editing Box Header

### DISCUSSION THREAD

**Discussion by andreibarabas** [GetStream/stream-chat-react-native#1185] [Ideas]
In the MessageInput, just like we can customise the InputButtons, we just need to make small tweaks to the "Reply to Message" header. Currently, we made a patch but that will soon break.

<img width="337" alt="Screenshot 2022-03-05 at 06 19 35" src="https://user-images.githubusercontent.com/2101405/156867416-ecb60a68-6552-4911-bc87-c4eb03bb2c8d.png">

**Comment by khushal87** [comment]
Hey, we have already released the feature here with the PR #1208.
You can refer to the PR description and customization docs - https://getstream.io/chat/docs/sdk/reactnative/guides/message-input-customization/#creating-a-custom-editing-state-header-in-messageinput
Thanks 😄

**Reply by andreibarabas** [reply]
thanks
